src/BloomBrowserUI/lib/long-press/jquery.longpress.js

```diff
@@ -164,13 +170,13 @@
         }
     }
     function showPopup(chars) {
-        popup.empty();
+        popup.find('ul').empty();
```

Comment:
See line 9, which does that. But really, we have gone far from the original now, and would like to go a lot farther.